src/app/(privateProfile)/analytics/page.tsx

```diff
@@ -1,8 +1,306 @@
+		},
+		{
+			id: 8,
+			background: "--color-dark-sea-green",
+			title: "Бензин",
+			valuePercent: "0.75",
+			valueRub: 975.56,
+		},
+		{
+			id: 9,
+			background: "--color-dusty-rose",
+			title: "Дорога работа",
+			valuePercent: "1.03",
+			valueRub: 1340.79,
+		},
+		{
+			id: 10,
+			background: "--color-steel-blue",
+			title: "Юрист",
+			valuePercent: "7.00",
+			valueRub: 9110.05,
+		},
+		{
+			id: 11,
+			background: "--color-sunny-yellow",
+			title: "Детский сад",
+			valuePercent: "12.40",
+			valueRub: 16192.09,
+		},
+		{
+			id: 12,
+			background: "--color-peach-pink",
+			title: "Учебники",
+			valuePercent: "2.00",
+			valueRub: 2600.01,
+		},
+		{
+			id: 13,
+			background: "--color-forest-green",
+			title: "Отпуск",
+			valuePercent: "4.95",
+			valueRub: 6437.57,
+		},
+		{
+			id: 14,
+			background: "--color-olive-green",
+			title: "Театр",
+			valuePercent: "1.30",
+			valueRub: 1690.01,
+		},
+		{
+			id: 15,
+			background: "--color-pale-lemon",
+			title: "Кредит",
+			valuePercent: "20.00",
+			valueRub: 26000.15,
+		},
+		{
+			id: 16,
+			background: "--color-dark-olive",
+			title: "Страховка",
+			valuePercent: "0.40",
+			valueRub: 520.0,
+		},
+		{
+			id: 17,
+			background: "--color-golden-olive",
+			title: "Киберспорт",
+			valuePercent: "0.40",
+			valueRub: 520.0,
+		},
+		{
+			id: 18,
+			background: "--color-light-blue-gray",
+			title: "Путешествия",
+			valuePercent: "0.40",
+			valueRub: 520.0,
+		},
+		{
+			id: 19,
+			background: "--color-burnt-orange",
+			title: "Кино",
+			valuePercent: "0.40",
+			valueRub: 520.0,
+		},
+		{
+			id: 20,
+			background: "--color-bright-lime",
+			title: "Ипотека",
+			valuePercent: "7.37",
+			valueRub: 9586.33,
+		},
+	];
+	const WINDOW_SIZE = 1440;
+	const MIN_ROW = 0;
+	const MAX_ROW = 8;
+
+	// const itemsToShow = window.innerWidth <= 1440 ? 0 : 8;
+	const [itemsToShow, setItemsToShow] = useState(MAX_ROW);
+	const [displayMode, setDisplayMode] = useState("rub");
+
+	useEffect(() => {
+		const handleResize = () => {
+			setItemsToShow(window.innerWidth <= WINDOW_SIZE ? MIN_ROW : MAX_ROW);
+		};
+
+		handleResize();
+
+		window.addEventListener("resize", handleResize);
+
+		return () => {
+			window.removeEventListener("resize", handleResize);
+		};
+	}, []);
+
+	const handleDisplayChange = (event: ChangeEvent<HTMLInputElement>) => {
+		setDisplayMode(event.target.value);
+	};
+
 	return (
-		<>
-			<div>Analytics</div>
-		</>
+		<div className={styles.analyticsPageWrap}>
+			<div className={styles.analyticsPageContainer}>
+				<h1 className={styles.headerTitle}>Аналитика</h1>
+
+				{/* <div className={styles.analyticsBlankPage}>
+
+					<p className={styles.analyticsBlankPage__text}>К сожалению, этот раздел пока пуст. Начните вести учет финансов в приложении и сможет воспользоваться этим разделом.</p>
+					
+					<MoneyIcon classNames={styles.analyticsBlankPage__image}/>
+
+				</div> */}
+
+				<div className={styles.analyticsPagesContent}>
```

Comment:
can you remake it by adding conditional rendering with mock control variable. For example:
const isEmptyPage = false;

...

return (

...
{
isEmptyPage ? <>empty page content</> : <>data page content</>
}
...

);